Neos.Flow/Classes/ResourceManagement/Target/FileSystemSymlinkTarget.php

```diff
@@ -32,18 +32,19 @@ class FileSystemSymlinkTarget extends FileSystemTarget
      * Publishes the whole collection to this target
      *
      * @param CollectionInterface $collection The collection to publish
-     * @param callable $callback Function called after each resource publishing
      * @return void
      */
-    public function publishCollection(CollectionInterface $collection, callable $callback = null)
+    public function publishCollection(CollectionInterface $collection)
     {
         $storage = $collection->getStorage();
         if ($storage instanceof PackageStorage) {
             foreach ($storage->getPublicResourcePaths() as $packageKey => $path) {
+                // FIXME we cannot invoke the callback here as we just symlink the whole folder,
```

Comment:
```suggestion
                // TODO we cannot invoke the callback here as we just symlink the whole folder,
```
TODO is potentially easier to find (by humans and tools alike)

Also, what could we potentially improve here?

Reply:
Fixed it by actually calling the callback so 

`flow resource:publish --collection static`

will show

```
Publishing resources of collection "static"
 - Published 11
```